page.go

```diff
@@ -76,7 +76,7 @@ type CreatePageParams struct {
 type UpdatePageParams struct {
 	// Either DatabasePageProperties or Title must be not nil.
 	DatabasePageProperties *DatabasePageProperties
-	Title                  []RichText
+	Archived               bool `json:"archived,omitempty"`
```

Comment:
Because `archived` is an optional param when updating a page ([ref](https://developers.notion.com/reference/patch-page)), we need to make this a `*bool`. Otherwise, you wouldn't be able to "unarchive" a page (because the empty value for `bool` (`false`) would get omitted from JSON.